routes/index.js

```diff
@@ -12,6 +12,15 @@ const { getSettings, postSettings } = require('./settings')
 const { getLandingPage } = require('./landingPage')
 const { getImprint } = require('./imprint')
 
+// Display settings in case of invalid credentials
+router.all('*', async (request, response, next) => {
+  if (response.locals.forceSettingsRoute) {
+    await getSettings(request, response, next)
```

Comment:
Can we do an `next('/settings')` here? 🤔

Reply:
nope if you pass anything to next it will be treated as an error

Reply:
we can do response.redirect('/settings')

Reply:
Which won't work because headers have been sent already.